plugins/node/opentelemetry-instrumentation-restify/package.json

```diff
@@ -57,13 +57,13 @@
     "restify": "4.3.4",
     "rimraf": "3.0.2",
     "ts-mocha": "10.0.0",
-    "typescript": "4.3.5"
+    "typescript": "4.3.5",
+    "@types/restify": "4.3.8"
```

Comment:
If you move the types to dev dependencies I believe users have compilation failures. That's why we moved them to regular dependencies to begin with.

Reply:
I'll take a look and see if that happens locally.